src/town_cmd.cpp

```diff
@@ -2721,16 +2721,17 @@ static bool CheckTownBuild2x2House(TileIndex *tile, Town *t, int maxz, bool nosl
  * @param hs The @a HouseSpec of the house.
  * @param house The @a HouseID of the house.
  * @param random_bits The random data to be associated with the house.
+ * @param completed Should the house be placed already complete, instead of under construction?
  */
-static void BuildTownHouse(Town *t, TileIndex tile, const HouseSpec *hs, HouseID house, uint8_t random_bits)
+static void BuildTownHouse(Town *t, TileIndex tile, const HouseSpec *hs, HouseID house, uint8_t random_bits, bool completed)
```

Comment:
```suggestion
static void BuildTownHouse(Town *t, TileIndex tile, const HouseSpec *hs, HouseID house, uint8_t random_bits, bool build_completed)
```

Or something. `completed` is very non-descriptive in this context :) `fully_constructed`? I dunno ...

Reply:
For what it's worth, `completed` is the word used elsewhere in the code. But I changed it to `house_completed`, hope that makes more sense to you. 🙂